libs/core/schematics/migrations/migration-0.36/index.ts

```diff
@@ -0,0 +1,129 @@
+}
+
+function removeIconFonts(): Rule {
+    return (tree: Tree, context: SchematicContext) =>
+        updateWorkspace((workspace) => {
+            workspace.projects.forEach((project) => {
+                project.targets.forEach((target) => {
+                    const styles = target.options?.styles;
+
+                    if (!target.options?.styles || !styles || !isJsonArray(styles)) {
+                        return;
+                    }
+
+                    const iconFonts = ['sap_fiori_3_fonts', 'sap_horizon_fonts'];
+
+                    iconFonts.forEach((fontFile) => {
+                        // Remove file
+                        const fontFileWithPath = `${project.sourceRoot}/theming/${fontFile}.css`;
+                        if (tree.exists(fontFileWithPath)) {
+                            tree.delete(fontFileWithPath);
+                            context.logger.info(`✅️ Removed ${fontFile}.css file.`);
+                        }
+
+                        // Remove from config
+                        const indexInStylesArray = styles.findIndex(
+                            (style) => isJsonObject(style) && style?.bundleName === `${fontFile}`
+                        );
+                        if (indexInStylesArray > -1) {
+                            styles.splice(indexInStylesArray, 1);
+                            context.logger.info(`✅️ Removed '${fontFile}' style from angular.json`);
+                        }
+                    });
+
+                    target.options.styles = styles;
+                });
+            });
+        });
+}
+
+function removeFontStyles(): Rule {
+    return (tree: Tree, context: SchematicContext) => {
+        const styleSheets = findStylesheetFiles(tree);
+
+        if (styleSheets.length > 0) {
+            const FONT_FACE_REGEX = /(@font-face)[\w\s]?{[s\S\W\w]+?(?=\s}\s)?\s?}[\n\s]*/gi;
+
+            styleSheets.forEach((styleSheet) => {
+                let buffer = tree.read(styleSheet);
+                if (!buffer) {
+                    return;
+                }
+
+                let contents = buffer.toString();
+
+                if (FONT_FACE_REGEX.test(contents)) {
+                    const matches = contents
+                        .match(FONT_FACE_REGEX)!
+                        .filter((m) => m.match(/~@sap-theming.*\.woff/)?.length);
+
+                    if (matches.length === 0) {
+                        return;
+                    }
+
+                    matches.forEach((match) => {
+                        contents = contents.replace(match, '');
+                    });
+
+                    tree.overwrite(styleSheet, contents);
+                    context.logger.info(`✅️ Removed font styles from stylesheets.`);
+                }
+            });
+        }
+
+        context.logger.info(
+            `⚠️ Notice: If you have imported font styles from @sap-theming, please remove it since now it controlled by fundamental-ngx`
+        );
```

Comment:
```suggestion
        context.logger.info(
            `⚠️ Notice: If you have imported font styles from @sap-theming, please remove it, since now it is controlled by fundamental-ngx`
        );
```